app/queries/servers/scheduled_post.ts

```diff
@@ -26,3 +26,47 @@ export const queryScheduledPostsForTeam = (database: Database, teamId: string, i
 export const observeScheduledPostsForTeam = (database: Database, teamId: string, includeDirectChannelPosts?: boolean) => {
     return queryScheduledPostsForTeam(database, teamId, includeDirectChannelPosts).observeWithColumns(['update_at']);
 };
+
+export const observeScheduledPostCount = (database: Database, teamId: string, includeDirectChannelPosts?: boolean) => {
+    return queryScheduledPostsForTeam(database, teamId, includeDirectChannelPosts).observeCount();
+};
+
+export const observeScheduledPostCountForChannel = (
+    database: Database,
+    teamId: string,
+    channelId: string,
+    isCRTEnabled: boolean,
+) => {
+    let query = database.collections.get<ScheduledPostModel>(SCHEDULED_POST).query(
+        Q.on(CHANNEL,
+            Q.and(
+                Q.where('team_id', teamId),
+                Q.where('id', channelId),
+            ),
+        ),
+    );
+
+    if (isCRTEnabled) {
+        query = query.extend(Q.where('root_id', ''));
+    }
+
+    return query.observeCount();
+};
+
+export const observeScheduledPostCountForDMsAndGMs = (database: Database, channelId: string, isCRTEnabled: boolean) => {
```

Comment:
by making the change above, this function is not needed

Reply:
Okay